src/lib.rs

```diff
@@ -684,12 +698,12 @@ type CategoryTreePath<CategoryId, ThreadId, BlockNumber, Moment> =
     Vec<Category<CategoryId, ThreadId, BlockNumber, Moment>>;
 
 decl_storage! {
-    trait Store for Module<T: Trait> as Forum {
+    trait Store for Module<T: Trait> as NewForum {
```

Comment:
must change it, I found the problem in unit test. When NextPostId is updated in old forum, then NextPostId in new forum also updated if don't change the name. I think in low level, storage name + variable name form the key in db.

Reply:
I see. But then we need some sort of naming scheme which has versioning in it, we cant come up with arbitrary names each time. Can we use digits in a name? Because then the natural approach would be `Forum_1_1` for example, for version 1.1 of the forum. If this is not allowed, we could use base64 encoding of the numbers, so it would be `Forum_B_G` for version 1.6 for example.

Reply:
I tried, Forum_1_1 is accepted by compiler. I will use it.